src/vsr/client.zig

```diff
@@ -276,13 +282,23 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
             if (was_empty) self.send_request_for_the_first_time(message);
         }
 
-        /// Acquires a message from the message bus if one is available.
-        pub fn get_message(self: *Self) *Message {
+        /// Acquires a message from the message bus.
+        ///
+        /// The caller must ensure that a message is available.
+        // FIXME: change back to get_message.
+        pub fn touch_get_message(self: *Self) *Message {
```

Comment:
Renamed just to have the CI yell at me if I missed a callsite, will rename back before merge.